code/mission/import/xwingmissionparse.cpp

```diff
@@ -623,6 +623,144 @@ void parse_xwi_flightgroup(mission *pm, const XWingMission *xwim, const XWMFligh
 	}
 }
 
+void parse_xwi_objectgroup(mission *pm, const XWingMission *xwim, const XWMObject *oj)
```

Comment:
Split off the xwingmissionparse changes into a separate branch; they should not be part of this PR.  We'll cross that bridge when we come to it.